rskj-core/src/main/java/co/rsk/peg/ErpRedeemScriptBuilderUtils.java

```diff
@@ -0,0 +1,35 @@
+package co.rsk.peg;
+
+import co.rsk.bitcoinj.core.NetworkParameters;
+import org.ethereum.config.blockchain.upgrades.ActivationConfig;
+import org.ethereum.config.blockchain.upgrades.ConsensusRule;
+
+public class ErpRedeemScriptBuilderUtils {
+
+    private ErpRedeemScriptBuilderUtils() {
+    }
+
+    public static ErpRedeemScriptBuilder defineNonStandardErpRedeemScriptBuilder(
+        ActivationConfig.ForBlock activations,
+        NetworkParameters networkParameters) {
+
+        ErpRedeemScriptBuilder erpRedeemScriptBuilder;
+
+        boolean networkParametersIsTestnetOrRegtest = checkIfNetworkParametersAreTestnetOrRegtest(networkParameters);
+
+        if(!activations.isActive(ConsensusRule.RSKIP284) && networkParametersIsTestnetOrRegtest) {
+            erpRedeemScriptBuilder = new NonStandardErpRedeemScriptBuilderHardcoaded();
+        } else if (!activations.isActive(ConsensusRule.RSKIP293)) {
+            erpRedeemScriptBuilder = new NonStandardErpRedeemScriptBuilderWithCsvUnsignedBE();
+        } else {
+            erpRedeemScriptBuilder = new NonStandardErpRedeemScriptBuilder();
+        }
+
+        return erpRedeemScriptBuilder;
+    }
+
+    public static boolean checkIfNetworkParametersAreTestnetOrRegtest(NetworkParameters networkParameters) {
```

Comment:
No need to check for regtest I believe. Only testnet

Reply:
It implies changing some tests. But no worries, its done. Just for you to know when you review